integration_tests/dbt_project.yml

```diff
@@ -11,6 +11,7 @@ models:
 
 vars:
   quickbooks_source:
+    using_purchase_order: True
```

Comment:
As the purchase order models are `false` by default, I decided to set the variable as `true` within the integration tests so the docs would still contain these models once we deploy to github pages.